oidc_example/op2/README.md

```diff
@@ -0,0 +1,58 @@
+OP2 Example
+===========
+
+Getting Started
+---------------
+
+    git clone https://github.com/OpenIDC/pyoidc.git
+    cd pyoidc/oidc_example/op2/
+    python3 -m venv venv && . venv/bin/activate
+    pip install -r requirements.txt
+
+
+### Client Management
+
+To be able to start up the project you have to create a client in
```

Comment:
Can you please reformat to one sentence per line? I think that keeps the file more readable.

Reply:
Fixed in 81d56dc!

Reply:
I see what you did there ... :)